src/views/dialogs/DepositDialog2/DepositForm.tsx

```diff
@@ -133,7 +187,10 @@ export const DepositForm = ({
           selectedSpeed={selectedSpeed}
           onSelectSpeed={setSelectedSpeed}
         />
-        <div tw="flex flex-col gap-0.5">
+        <div
+          tw="flex flex-col gap-0.5"
+          style={{ opacity: depositSteps ?? awaitingWalletAction ? '0.5' : '1' }}
```

Comment:
style and disabled conditionals are the same, can consolidate

Reply:
great call